email_templates/plaintext/decom_email.j2

```diff
@@ -24,9 +25,9 @@ Where possible we will not decommission hardware that is still hosting User owne
 if the<br>Hypervisor hosting your VM fails we have NO spares and will NOT attempt anything beyond basic repair work,
 <br>in this case you will suffer the total loss of your VM.
 
-Failure to comply with this notice will eventually lead to Hardware being turned off and disposed of, whether it<br>
-is still hosting VMs or not, eventually we will have no choice. Routine reminders will follow and increase in
-frequency over the coming weeks.
+Failure to comply with this notice by mid day (12:00PM) on Wednesday 28th February 2024 will lead to Hardware being 
+turned off and disposed of, whether it<br> is still hosting VMs or not, we have no choice at that point. Routine 
```

Comment:
There's a `<br>` which I think is from the HTML version of the message

Reply:
Thanks! Fixed in latest commit